test/spec/pkg-spec.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * Test existance of all needed files in each package.
+ * This test file should be run before each release.
+ */
+
+const fs = require('fs');
+const path = require('path');
+const SpecReporter = require('jasmine-spec-reporter').SpecReporter;
+
+// We use 'jasmine-spec-reporter' to get nice logs to console.
+jasmine.getEnv().clearReporters();               // remove default reporter logs
+jasmine.getEnv().addReporter(new SpecReporter({  // add jasmine-spec-reporter
+  spec: {
+    displayPending: true // display each pending spec
+  },
+}));
+
+/**
+ * Get path of given fileName in given pkgName
+ * @param pkgName
+ * @param fileName
+ * @returns {*}
+ */
+const getPath = (pkgName, fileName) => path.join(__dirname, '..', '..', 'pkg', pkgName, fileName);
+
+
+/**
+ * Test that a given fileName exist in given pkgName
+ * @param pkgName
+ * @param fileName
+ */
+testFileExists = (pkgName, fileName) => {
```

Comment:
More appropriate name seems to be assertPathExists (in any case you check both files and dirs)